packages/plugin-essentials/sources/commands/install.ts

```diff
@@ -68,8 +73,9 @@ export default class YarnCommand extends BaseCommand {
     description: `Verbosely print the output of the build steps of dependencies`,
   });
 
-  skipBuilds = Option.Boolean(`--skip-builds`, false, {
```

Comment:
Since the Yarn is in pre-release for Yarn 3, `--skip-builds` is removed instead of deprecation, right?

Reply:
Yep - we could transform it into `--mode` for a major, but given that's a relatively fringe flag (especially since we don't really recommend using it, people should usually use `enableScripts` instead), it's probably fine to remove it altogether.